src/hooks/useSwapAdjustedAmounts.ts

```diff
@@ -0,0 +1,48 @@
+import { Trade } from '@uniswap/sdk';
+import { useSelector } from 'react-redux';
+import {
+  computeSlippageAdjustedAmounts,
+  Field,
+} from '@rainbow-me/handlers/uniswap';
+import { AppState } from '@rainbow-me/redux/store';
+import { SwapModalField } from '@rainbow-me/redux/swap';
+import { updatePrecisionToDisplay } from '@rainbow-me/utilities';
+
+export default function useSwapAdjustedAmounts(tradeDetails: Trade) {
+  const genericAssets = useSelector(
```

Comment:
isn't this code copy-pasted?

Reply:
I'm not sure what you mean - I'm reusing the computeSlippageAdjustedAmounts but we have not displayed the minimum amount received/maximum amount sold anywhere.